added new exercise: camicia (#2581)
exercism/problem-specifications: exercises/camicia/introduction.md

# Introduction

One rainy afternoon, you sit at the kitchen table playing cards with your grandmother.

> The game is [Camicia][bmn].

Reviewer: ```suggestion
The game is her take on [Camicia][bmn].
```

At first it feels like just another friendly match: cards slapped down, laughter across the table, the occasional victorious grin from Nonna.

> But as the game drags on, something strange happens.

Reviewer 2: "the game __drags__ on"

Sounds depressing, like you do not want to be there.

Author: "But as the game stretches on, something strange happens."

Any suggestions?
I am not a native English speaker, nor am I an expert AHAH

Reviewer 2: The "game stretches on" is less depressing to me than something that just "drags on".

A thesaurus can be helpful though.

> The pile keeps growing.

Reviewer 2: How many decks are we playing with?  Does the pile keep growing?  Or does it keep cycling?

Author: "The same cards keep cyclingback. You play card after card, yet the end never seems to arrive."

Is that alright?

You play card after card, yet the end never seems to come.

You start to wonder.
_Will this game ever finish?
Or could we keep playing forever?_

> Later, driven by curiosity, you search online and to your surprise you discover that what happened wasn’t just bad luck.

Reviewer: ```suggestion
Later, driven by curiosity, you search online and to your surprise you discover that what happened wasn't just bad luck.
```

# Introduction

One rainy afternoon, you sit at the kitchen table playing cards with your grandmother.\

> The game is [Beggar-my-neighbour][bmn], or as she calls it, “Camicia.” 

Reviewer 3: ```suggestion
The game is [Beggar-my-neighbour][bmn], or as she calls it, “Camicia.”
```

At first it feels like just another friendly match:\
cards slapped down, laughter across the table, the occasional victorious grin from Nonna.\
But as the game drags on, something strange happens.\
The pile keeps growing.\
You play card after card, yet the end never seems to come.

You start to wonder:\
_Will this game ever finish? Or could we keep playing forever?_

Later, driven by curiosity, you search online and to your surprise you discover that what happened wasn’t just bad luck.\
You and your grandmother may have stumbled upon one of the longest possible sequences!\
Suddenly, you’re hooked.\
What began as a casual game has turned into a quest:\
_How long can such a game really last?_\
_Can you find a sequence even longer than the one you played at the kitchen table?_\
_Perhaps even long enough to set a new world record?_

> And so, armed with nothing but a deck of cards and some algorithmic ingenuity, you decide to investigate…

Reviewer 2: Why not the ellipse?  Just wondering, it is a proper character, after all.

> And so, armed with nothing but a deck of cards and some algorithmic ingenuity, you decide to investigate…

Reviewer: I believe it's the only character in the file which cannot be found on most keyboards.

> [bmn]: https://en.wikipedia.org/wiki/Beggar-my-neighbour

Reviewer 3: ```suggestion
[bmn]: https://en.wikipedia.org/wiki/Beggar-my-neighbour

```
